### DISCUSSION TITLE
Functions Wiki Page Missing Content

### DISCUSSION THREAD

**Discussion by OneGameDad** [codam-coding-college/MLX42#145] [General]
The Functions page ([https://github.com/codam-coding-college/MLX42/wiki/Functions](https://github.com/codam-coding-college/MLX42/wiki/Functions)) which is linked from the table of contents is entirely empty. 

Do you want it filled out or removed?  (BTW, I did not file this under issues as it's not related to the code).

**Comment by W2Wizard** [comment]
<img width="578" alt="image" src="https://github.com/user-attachments/assets/80054692-e665-406f-a1c0-9037a56e41e1" />

@orezek Any reason why you decided to destroy an entire page?

**Comment by W2Wizard** [comment]
I keep these wiki pages open to edit so anyone can fix / suggest things but if I see someone completely delete a page thats a bit rude.
